### DISCUSSION TITLE
How to make my minimalist-ui dashboard , the ONLY dashboard?

### DISCUSSION THREAD

**Discussion by gsotiriou** [UI-Lovelace-Minimalist/UI#1271] [Q&A] [answered]
Downloaded minimalist-ui installed following the instructions , plugged in the example from the documentation. All nice and cozy but on the sidebar I have the initial "Overview" from HA , and another entry labeled Minimalist UI. How can I make whatever configuration I have (through minimalist UI) my main dashboard (i.e. Overview) and getting rid of the "Minimalist UI" entry on my sidebar?

**Comment by modem158** [comment]
You can "hide" unwanted dashboards in the left panel by long pressing the "Home Assistant" name at the top of the list.

**Comment by modem158** [comment, marked as answer]
Also in your profile at the bottom of the left panel, clock on your name and choose default dashboard in the settings

**Reply by gsotiriou** [reply]
Thank you again!
Actually this is a more correct answer as the previous one only hides the other dashbooard but the client still defaults to the 'Overview'. I'll mark this one as correct.
